awx_collection/tools/vars/aliases.yml

```diff
@@ -0,0 +1,30 @@
+---
+aliases:
+  job_templates:
+    ask_tags_on_launch:
+      - ask_tags
+    ask_verbosity_on_launch:
+      - ask_verbosity
+    ask_diff_mode_on_launch:
+      - ask_diff_mode
+    allow_simultaneous:
+      - concurrent_jobs_enabled
+    diff_mode:
+      - diff_mode_enabled
+    ask_inventory_on_launch:
+      - ask_inventory
+    limit:
+      - ask_limit
+    force_handlers:
+      - force_handlers_enabled
+    ask_job_type_on_launch:
+      - ask_job_type
+    ask_skip_tags_on_launch:
+      - ask_skip_tags
+    use_fact_cache:
+      - fact_caching_enabled
+    extra_vars:
+      - ask_extra_vars
+    ask_credential_on_launch:
+      - ask_credential
```

Comment:
So this might be covering the answer to the prior issue I raised. I'm fine if it works. The concern is only backward compatibility.

Reply:
I still don't see support for things like `vault_credential`, and the worry is that users will download the collection, it won't run their playbook, and they rage quit.

Processing params like these old credential types goes against the current (really good) testing and maintenance philosophy. But to sell this to people, I see benefit from maintaining those for a little while longer. Was just talking to @wenottingham about these topics, and this reminds me that, in practice, the backward compatibility timeframe for the collection is a ridiculously absurdly long time. Historically, the modules have not been able to remove things in a reasonable period of time, and it paralyzed maintenance work. At this very moment, however, I can easily envision a short snippet that munges vault credential into the credentials list.